packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -352,6 +354,13 @@ class _CircularProgressIndicatorPainter extends CustomPainter {
       ..color = valueColor
       ..strokeWidth = strokeWidth
       ..style = PaintingStyle.stroke;
+    if (backgroundColor != null) {
+      final Paint backgroundPaint = Paint()
```

Comment:
I created a new `backgroundPaint` because this way we can test it with `paints..arc`.